hexa/pipelines/models.py

```diff
@@ -206,6 +209,17 @@ def update_if_has_perm(self, principal: User, **kwargs):
             if key in kwargs:
                 setattr(self, key, kwargs[key])
 
+        if "recipientIds" in kwargs:
+            PipelineRecipient.objects.filter(
+                Q(pipeline=self) & ~Q(user_id__in=kwargs["recipientIds"])
+            ).delete()
+            for id in kwargs["recipientIds"]:
```

Comment:
```suggestion
            for member in WorkspaceMembership.objects.filter(pipeline=self, user_id__in=kwargs["recipientIds"]):
                PipelineRecipient.objects.get_or_create(user=member.user, pipeline=self)
```

Reply:
You mean  
 ```python
 WorkspaceMembership.objects.filter(workspace=self.workspace, user_id__in=kwargs["recipientIds"]):
```
(field pipeline doesn't exist on WorkspaceMembership)?

Reply:
Yes, I just typed it in Github without checking what I was doing

Reply:
Okay. That's done.